.github/workflows/release.yml

```
@@ -0,0 +1,177 @@
+
+      - name: Set build variables
+        run: |
+          echo "VERSION=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+          echo "BUILD_DATE=$(date -u +%Y-%m-%dT%H:%M:%SZ)" >> $GITHUB_ENV
+          echo "GIT_COMMIT=${GITHUB_SHA}" >> $GITHUB_ENV
+
+      - name: Build binary
+        env:
+          GOOS: ${{ matrix.goos }}
+          GOARCH: ${{ matrix.goarch }}
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2089:warning:6:10: Quotes/backslashes will be treated literally. Use an array

(shellcheck)